erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -868,30 +868,34 @@ def make_inter_company_sales_order(source_name, target_doc=None):
 
 	return make_inter_company_transaction("Purchase Order", source_name, target_doc)
 
-
 @frappe.whitelist()
 def make_subcontracting_order(source_name, target_doc=None, save=False, submit=False, notify=False):
-	target_doc = get_mapped_subcontracting_order(source_name, target_doc)
-
-	if (save or submit) and frappe.has_permission(target_doc.doctype, "create"):
-		target_doc.save()
-
-		if submit and frappe.has_permission(target_doc.doctype, "submit", target_doc):
-			try:
-				target_doc.submit()
-			except Exception as e:
-				target_doc.add_comment("Comment", _("Submit Action Failed") + "<br><br>" + str(e))
-
-		if notify:
-			frappe.msgprint(
-				_("Subcontracting Order {0} created.").format(
-					get_link_to_form(target_doc.doctype, target_doc.name)
-				),
-				indicator="green",
-				alert=True,
-			)
+	is_sco_full = all(po_item.qty == po_item.sco_qty for po_item in frappe.get_doc("Purchase Order", source_name).items)
```

Comment:
Only qty and sco_qty is required here from the purchase order item table then why get_doc used, maybe get the list using qb where qty != sco_qty